workflow-runtime/src/main/java/com/squareup/workflow/RenderWorkflow.kt

```diff
@@ -163,19 +163,9 @@ fun <PropsT, OutputT : Any, RenderingT> renderWorkflowIn(
         // After receiving an output, the next render pass must be done before emitting that output,
         // so that the workflow states appear consistent to observers of the outputs and renderings.
         renderingsAndSnapshots.value = runner.nextRendering()
-        output?.let { onOutput(it) }
+        output.withValue { onOutput(it) }
       }
   }
 
   return renderingsAndSnapshots
 }
-
-/**
- * If this is already a [CancellationException], returns it as-is, otherwise wraps it in one with
- * the given message.
- */
-private inline fun Throwable?.toCancellationException(message: () -> String) = when (this) {
```

Comment:
This was already unused?

Reply:
Yea, it stopped being used in an earlier PR but the deletion got left out of a rebase.